tests/test_files/libcpp_stl_test.hpp

```diff
@@ -207,4 +207,14 @@ class LibCppSTLTest {
             }
             return -1;
         }
+        
+        int process_15_map(std::map<IntWrapper, IntWrapper>& in)
+        {
+            if (!in.empty())
+            {
+                in.begin()->second.i_ += 10;
+                return 1;
```

Comment:
can we be sure that returning 1 is sufficient for the test?
I briefly looked into the other tests and saw that they e.g., return the content of a variable (e.g., in.begin()->second.i_)